src/object/notification.c

```diff
@@ -50,16 +50,6 @@ static inline void maybeDonateSchedContext(tcb_t *tcb, notification_t *ntfnPtr)
         }
     }
 }
-
-static inline void maybeReturnSchedContext(notification_t *ntfnPtr, tcb_t *tcb)
-{
```

Comment:
I don't understand this change. The "previous commit" just adds an if? I'm confused.

Nit: it's nice if instead of 'previous commit' you use the hash, alhough this is a pain for rebasing.

Reply:
I thibnk github is confusing me - I'm not sure which commit "previous" is. But if its the other commit in order then this is commit is not relevant any more?